.github/workflows/integration-tests.yml

```diff
@@ -1,5 +1,9 @@
 name: Integration Tests
-on: [pull_request]
+on: 
+  pull_request:
+    paths:
+      - '*/**'
+      - '!contracts/**'
```

Comment:
Integration tests involve the entire repository, not sure how safe this is.

Reply:
@kalverra I _think_ we can ignore running the integration tests if only a contract changes (no core files), right? We can also just let this be triggered for all files and avoid trying to filter too.